content/zh-cn/docs/tasks/tools/install-kubectl-windows.md

```diff
@@ -55,7 +55,7 @@ The following methods exist for installing kubectl on Windows:
 ### 用 curl 在 Windows 上安装 kubectl {#install-kubectl-binary-with-curl-on-windows}
 
 <!--
-1. Download the [latest release {{< param "fullversion" >}}](https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe).
+1. Download the [latest release {{< skew currentPatchVersion >}}](https://dl.k8s.io/release/v{{< skew currentPatchVersion >}}/bin/windows/amd64/kubectl.exe).
```

Comment:
Line 57-59 can be removed, which is a dup of line 60-63.

Reply:
Thanks, @windsonsea. That change is outside the scope of this PR which is to simply replace the obsolete attribute wherever it appears. Could you create a separate issue/PR to handle the removal?

Reply:
Ok, marked and will do.